README.md

```diff
@@ -59,75 +59,66 @@ Javascript version:
 
 ## Features
 
```

Comment:
I still prefer to use the old way for this part, can you revert this part? thanks.

Reply:
Hello! I reverted it. Thank you for the quick response.